crates/ruff/src/directives.rs

```diff
@@ -107,15 +107,19 @@ pub fn extract_isort_directives(lxr: &[LexResult]) -> IsortDirectives {
         // omit a space after the colon. The remaining action comments are
         // required to include the space, and must appear on their own lines.
         let comment_text = comment_text.trim_end();
-        if comment_text == "# isort: split" {
+        if comment_text == "# isort: split" || comment_text == "# ruff: isort: split" {
             splits.push(start.row());
-        } else if comment_text == "# isort: skip_file" || comment_text == "# isort:skip_file" {
+        } else if comment_text == "# isort: skip_file"
+            || comment_text == "# isort:skip_file"
+            || comment_text == "# ruff: isort: skip_file"
+            || comment_text == "# ruff: isort:skip_file"
+        {
```

Comment:
Nit

```suggestion
        } else if matches!(comment_text, "# isort: skip_file" | "# isort:skip_file" | "# ruff: isort: skip_file" | "# ruff: isort:skip_file")
        {
```

or use a `match` statement 

```rust
match comment_text {
	"# isort: split" | "# ruff: isort: split" => { ... },
	"# isort: skip_file" | "# isort:skip_file" | " #ruff: isort: skip_file" | ... => {},
	_ if off.is_some() => {...
}
```